dev-docs/bidder-adaptor.md

```diff
@@ -12,259 +11,504 @@ nav_section: adapters
-| `bidderCode` | Required  | The bidder code.                                                         | `"pubmatic"`                         |
-| `cpm`        | Required  | The bid price. We recommend the most granular price a bidder can provide | 3.5764                               |
-| `width`      | Required  | The width of the returned creative.                                      | 300                                  |
-| `height`     | Required  | The height of the returned creative.                                     | 250                                  |
-| `ad`         | Required  | The creative payload of the returned bid                                 | `"<html><h3>I am an ad</h3></html>"` |
+If possible, we recommend that publishers associate a renderer with their outstream video ad units.  By doing so, all video-enabled demand partners will be able to participate in the auction, regardless of whether a given demand partner provides a renderer on its bid responses.  Prebid.js will always invoke a publisher-defined renderer on a given ad unit.
 
-## Step 6: Register User Sync Pixels
+However, if the publisher does not define a renderer, you will need to return a renderer with your bid response if you want to participate in the auction for outstream ad unit.
 
-In order to protect page performance, user ID sync logic within Prebid adapters should
-be queued using the userSync.registerSync() function. Both image pixels and iframe sync methods are supported, but
-publishers may control which method(s) are allowed.
+You can check for the outstream video context in your adapter as shown below, and then modify your response as needed to provide a renderer:
 
-At some point after the auctions are complete, Prebid will write out the registered userSyncs. See the userSync section of the Publisher API Reference for information about how Publishers can control which syncs are allowed.
+```javascript
+const videoMediaType = utils.deepAccess(bid, 'mediaTypes.video');
+const context        = utils.deepAccess(bid, 'mediaTypes.video.context');
 
-Add a user sync in an adapter in two steps:
+if (bid.mediaType === 'video' || (videoMediaType && context !== 'outstream')) {
+    /* Change this for your adapter as needed to provide a renderer. */
+}
+```
 
-1. Load the userSync module in the adapter file
-2. Register the sync
+## Supporting Native
 
-{% highlight js %}
-import { userSync } from 'src/userSync.js';
-
-   // ... code ...
-   if ($$PREBID_GLOBAL$$.userSync.iframeEnabled) {
-     userSync.registerSync('iframe', 'exampleAdapter', 'http://example.com/iframe-url');
-   } else {
-     userSync.registerSync('image', 'exampleAdapter', 'http://example.com/pixel');
-   }
-   // ... code ...
-{% endhighlight %}
+In order for your bidder to support the native media type, you need 2 things:
+
+1. Your (server-side) bidder needs to respond with a bid that has native information.
```

Comment:
Instead of "native information", we should always refer to "native assets".  We should also note that native bidders must be capable of ingesting the required and optional native assets specified on the `adUnit.mediaTypes.native` object.

Reply:
Addressed in a7a422e, specifically:

- Always refer to native *assets*

- Draw a clearer line between the server-side bidder's response and the (locally running) bidder adapter's response to Prebid.js

- Add a note that the adapter must be capable of ingesting native assets on `adUnit.mediaTypes.native`, and link to 'Show Native Ads' page for more info